test/e2e/network_test.go

```diff
@@ -38,6 +42,8 @@ var (
 	retryCount = 20
 	// retryInterval is the interval of time until we retry after a failure
 	retryInterval = 5 * time.Second
+	// testNamespace is the namespace to deploy our test pods on
+	testNamespace = "wmco-test"
```

Comment:
For one this is actually a constant. Moreover I suggest this becomes a property of the test context. What it is called is immaterial to the user. 

The name `testNamespace` is also a bit confusing as we also describe `testContext.namespace` as a test namespace. How about `workloadNamespace`?